.github/workflows/ci.yml

```diff
@@ -60,9 +60,7 @@ jobs:
           name: Logfiles
           path: |
             ./e2e_tests/bobtimus.log
-            ./e2e_tests/liquid.log
```

Comment:
Are those logs not used or not helpful; why remove them?

Reply:
These files simply do not exist :D